src/conduit-lwt-unix/conduit_lwt_unix.mli

```diff
@@ -166,7 +166,8 @@ val init :
 (** [init ?src ?tls_own_key ()] will initialize a Unix conduit that binds to the
     [src] interface if specified. If TLS server connections are used, then
     [tls_server_key] must contain a valid certificate to be used to advertise a
-    TLS connection *)
+    TLS connection. The certificate is validated against the OS trust anchors
```

Comment:
Can't we pass through the `authenticator` function here as an optional arg to init? That would allow it to be overridden for custom CAs, but the default would work for public anchors.